exercise_perseus_render/assets/src/vue/perseus.vue

```diff
@@ -454,4 +437,11 @@
   .perseus-hint-label:after
     content: ')'
 
+  .paragraph
+    padding: 4px
+
+  .svg-image *
+    padding-bottom: 0 !important
+
+
```

Comment:
looks good - thank you!

I updated this to use just a single nested selector using stylus syntax here: https://github.com/learningequality/kolibri-exercise-perseus-plugin/pull/10/commits/e4159619f038ac58cc46630078f11bfbaa31fdc8#diff-d7cf7d5c4d2bda00f84c872353b21af9